### ISSUE TITLE
Tensor.detach()の仕様について

### ISSUE THREAD

**Issue by CURRY-AND-RICE** [oreilly-japan/deep-learning-from-scratch-4#13]
pytorch版の[dqn.py](https://github.com/oreilly-japan/deep-learning-from-scratch-4/blob/master/pytorch/dqn.py)の83行目で、ターゲットネットワークを計算グラフから切り離す意図のコードがあると思いますが、
ソースコード中の
```python3
next_q.detach()
```
は
```python3
next_q = next_q.detach()
```
の間違いではないでしょうか。
